language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/signature/SignatureHelpUtil.java

```diff
@@ -424,8 +424,14 @@ private static SignatureInfoModel getSignatureInfoModel(BInvokableSymbol bInvoka
         if (restParam != null) {
             parameters.add(new Parameter(restParam.name.value, restParam.type, false, true));
         }
+        boolean isLangLib = CommonUtil.isLangLibSymbol(bInvokableSymbol);
```

Comment:
This will capture all the langlib function symbols. I think here we need to capture the following case as well, where we need to show the hint for the first parameter, shall we cover that particular case well?
`var s = 'int:abs(<cursor>)`

Reply:
+1, this was not covered. will work on it

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/25135/commits/502b66345968d62f46b0686111873439e41f57e2